pkg/cmd/oauth-server/cmd.go

```diff
@@ -19,24 +20,27 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
-type OsinServer struct {
+type OsinServerOptions struct {
```

Comment:
I changed the name of the struct to reflect that it actually contain the options for the server. It is now more similar to oauth-apiserver.